clients/java/src/main/java/HubConnection.java

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
`clients/java/src/main/java/filename.java` seems a bit much, maybe `clients/java/src/filename.java`?

Reply:
`src/main/java` is the standard convention for Java projects